packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -154,7 +181,7 @@ export const wrapCommand = (command) => {
       // TODO: Gauntlet core should initialize commands using `buildCommand` instead of new Command
       await this.buildCommand(this.flags, this.args)
 
-      let proposalId = Number(this.flags.proposal || this.flags.multisigProposal) // alias requested by eng ops
+      let proposalId = Number(this.flags.proposal || this.flags.multisigProposal)
```

Comment:
These comments had a purpose:

- Point to alias as an external feature request
- Point that aliases are not explicitly supported by Gauntlet and are added ad-hoc

I'd suggest tracking this somewhere and figuring out how aliases fit the bigger picture and how are they handled by the framework. I'm personally against any kind of aliases at this point as they bring more confusion than clarity.